source/clear-linux/tutorials/smb/smb-desktop.rst

```diff
@@ -0,0 +1,75 @@
+.. _smb-desktop:
+
+Connect to Windows\* share from |CL-ATTR| desktop
+#################################################
+
+This tutorial explains how to access an existing shared location on Windows\* via SMB in the |CL-ATTR| GNOME GUI desktop.
```

Comment:
A sentence explaining Samba and the SMB acronym would be good.

Reply:
Done.